heudiconv/utils.py

```diff
@@ -495,7 +521,11 @@ def create_tree(path, tree, archives_leading_dir=True):
             executable = False
             name = file_
         full_name = op.join(path, name)
-        if isinstance(load, (tuple, list, dict)):
+        if name.endswith('.json') and isinstance(load, dict):
+            # (For a json file, we expect the content to be a dictionary, so
+            #  don't continue creating a tree, but just write dict to file)
+            save_json(full_name, load, indent=4)
```

Comment:
```suggestion
            save_json(full_name, load)
```
so we just stay consistent with our current/default behavior across files.